pkgs/applications/editors/vscode/extensions/vadimcn.vscode-lldb/default.nix

```diff
@@ -106,7 +106,9 @@ in stdenv.mkDerivation {
 
     mkdir -p $ext/{adapter,formatters}
     mv -t $ext vsix-extracted/extension/*
-    cp -t $ext/adapter ${adapter}/{bin,lib}/* ../adapter/*.py
+    cp -t $ext/adapter ${adapter}/{bin,lib}/*
+    cp -r ../adapter/scripts $ext/adapter
+    rm $ext/adapter/CMakeLists.txt
```

Comment:
This file does not seem to exist.
https://github.com/NixOS/nixpkgs/pull/240445/checks?check_run_id=14641969218

Reply:
Good eye, will fix shortly.